kong/plugins/aws-lambda/handler.lua

```diff
@@ -83,7 +83,11 @@ function AWSLambdaHandler:access(conf)
     return responses.send_HTTP_INTERNAL_SERVER_ERROR(err)
   end
 
-  ngx.status = res.status
+  if headers['X-Amzn-Function-Error'] == 'Unhandled' and conf.unhandled_response ~= -1 then
```

Comment:
style: we use double quotes in this codebase, and try to avoid having lines over 80 characters long. See our style guide: https://github.com/Mashape/kong/blob/master/CONTRIBUTING.md#code-style